src/components/utils/messages.js

```diff
@@ -6,9 +6,8 @@ const intlData = {
         unexpectedError:
             "This wasn't supposed to happen. Please try again or contact support!",
         signInReqd: "Sign In Required",
-        signInPrompt: "You must sign in to view this page.",
+        signInPrompt: "You must sign in to perform this action.",
```

Comment:
I thought this message applied to more situations, including navigating to another page that requires sign-in.